chess2.cpp

```diff
@@ -0,0 +1,882 @@
+#include "chess2.h"
+#include<iostream>
+
+int Chess2::perform(str Move) {
+    int original;
+    original = board[Move[2] - 48][Move[3] - 48];
+    board[Move[2] - 48][Move[3] - 48] = board[Move[0] - 48][Move[1] - 48];
+    board[Move[0] - 48][Move[1] - 48] = 0;
+    return original;
+}
+
+str Chess2::push(int row, int col, int desrow, int descol) {
```

Comment:
incorrect naming of the function as it is simply concatenating different rows and cols to a string rep..